__tests__/endpoints.test.js

```diff
@@ -230,6 +230,68 @@ describe("POST: /api/articles/:article_id/comments", () => {
   });
 });
 
+describe("PATCH: /api/articles/:article_id", () => {
+  test("PATCH: 200 respond with the corresponding article object when decrementing votes", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: -60 })
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article.article_id).toBe(1);
+        expect(body.article).toMatchObject({
+          article_id: expect.any(Number),
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          article_img_url: expect.any(String),
+        });
+      });
+  });
+
+  test("PATCH: 200 respond with the corresponding article object when incrementing votes", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: 60 })
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article.article_id).toBe(1);
+        expect(body.article).toMatchObject({
+          article_id: expect.any(Number),
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          article_img_url: expect.any(String),
+        });
```

Comment:
This assertions doesn't check that the votes property goes up by 60. Can you update it to check for an exact votes property.